crates/sound/src/lib.rs

```diff
@@ -223,6 +230,10 @@ impl SoundConfig {
     pub fn get_socket_path(&self) -> String {
         String::from(&self.socket)
     }
+
+    pub fn get_audio_backend(&self) -> &BackendType {
+        &self.audio_backend
+    }
```

Comment:
```suggestion
    pub fn get_audio_backend(&self) -> BackendType {
        self.audio_backend
    }
```
It is a simple enum, with Copy derived you can drop the `&

Reply:
Or explicitely calling `self.audio_backend.clone()`. Not sure which option to favour in this case :shrug:

Reply:
It should be a copy type.